### ISSUE TITLE
harmonize ID location

### ISSUE THREAD

**Issue by dasch124** [acdh-oeaw/vicav-app#101]
With the introduction of unmonitored speech in TUNOCENT, we have two places where IDS are stored: 

In the "old" VICAV data types, the IDs consumed by the backend are `@xml:id`s on `<div>` elements, while in the transcription documents, these are `<idno type="{project-acronym}CorpusID">`. Obviously this is far from ideal and we should harmonize this. I see two aspects here: 
* The scope of `@xml:id`s is limited to the document where they are stored, so using it for dataset-wide identifiers is somehow a stretch (although an admittedly common one) 
* Actually all VICAV-compatible data should end up in VICAV production, so this would be our target scope for identity. 

All in all, I'd propose to store identifiers of all VICAV data types inside of `<idno type="VICAV">…</idno>` . 

Thoughts? Counter proposals? @simar0at @charlymo @MauPalantir @VeronikaEngler

**Comment by MauPalantir** [comment]
Yes, the idno is more explicit. Just have a standard master idno flag, and not something that has to be identified with a starts-with()

**Comment by dasch124** [comment]
> Just have a standard master idno flag, and not something that has to be identified with a starts-with()

Right, that's why I thought of "VICAV".
